superset-frontend/src/dashboard/stylesheets/components/tabs.less

```diff
@@ -46,7 +46,7 @@
         }
       }
 
-      & .dragdroppable-tab {
+      & .dragdroppable-tab[draggable='true'] {
```

Comment:
Why was this not a problem before? Was `dragdroppable-tab` dynamically added but somehow it's not possible to do so in the new implementation?

Reply:
Well, it's all a big pile of overrides!

The prior situation was "cursor:move" all the way down to the tab text, which _was_ an `<input type="button">` which had an override with `cursor:pointer` via this super simple selector:
`.dashboard-component-tabs .nav-tabs>li .editable-title input[type=button]`

Instead of a button, that tab text is now in a `span`, so the override is not being applied. It happened [here](https://github.com/apache/incubator-superset/blob/17e9f6d23b96d94d8f86f409247c9d90000cc358/superset-frontend/src/components/EditableTitle.tsx#L190-L197) via #10936